Amazon.QLDB.Driver/driver/QldbDriverBuilder.cs

```diff
@@ -188,7 +188,7 @@ internal static IRetryHandler CreateDefaultRetryHandler(ILogger logger)
         private static void SetUserAgent(object sender, RequestEventArgs eventArgs)
         {
             const string UserAgentHeader = "User-Agent";
-            const string Version = "1.1.0";
+            const string Version = "1.1.1";
```

Comment:
All other looks good. But can you move the revision update to a separate PR?

Reply:
Done